pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -798,13 +798,16 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 			messageBuilder: func() [][]byte {
 				httpMethods = []string{http.MethodGet, http.MethodPost, http.MethodHead, http.MethodDelete,
 					http.MethodPut, http.MethodPatch, http.MethodOptions, http.MethodTrace, http.MethodConnect}
+				httpMethods = append(httpMethods, httpMethods...)
```

Comment:
not sure why it is necessary? can't we use only one list or create one list that contains all needed methods

Reply:
added a comment